packages/frontend/src/features/videos/providers/Slide.tsx

```diff
@@ -27,7 +27,9 @@ class Slide extends React.Component<SlideProps, SlideState> {
     margin: 0,
   };
 
-  private handleResize = () => {
+  private _isAnimating: boolean = false;
```

Comment:
w ts nie stosuje sie _przed nazwa pola bo w js mialo to zastępować hermetyzacje którą ts już posiada